inf-terraform-aws/files/Makefile

```diff
@@ -0,0 +1,317 @@
+NAME                     := AWS Prototype
+DESCRIPTION              := The '$(NAME)' is a is a prototype for an ODS quickstarter
+VERSION                  := $(shell cat ./VERSION | tr -d [:space:])
+
+GIT_REMOTE_ORIGIN_URI    := $(shell git remote get-url origin)
+GIT_REMOTE_UPSTREAM_NAME := upstream
+GIT_REMOTE_UPSTREAM_URI  := ssh://git@bitbucket.biscrum.com:7999/infiaas/stack-aws-skeleton.git
+
+RELEASE_EMAIL_FROM       := zzITINFCoEforCloudComputing@boehringer-ingelheim.com
+RELEASE_EMAIL_TO         := cb4f6d49.boehringer.onmicrosoft.com@emea.teams.ms
```

Comment:
?

Reply:
The quickstarter replaces the need for an upstream stack framework, since it is integrated into the quickstarter, which is now the source of truth. Release emails should be empty initially and documented as convenience in README.md. Please make sure that the user experience does not break if emails are undefined.

Reply:
removed